.github/workflows/publish-and-release.yaml

```diff
@@ -0,0 +1,98 @@
+#
+# Copyright (c) 2021 - for information on the respective copyright owner
+# see the NOTICE file and/or the repository https://github.com/carbynestack/common.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+name: Publish and release
+on:
+  workflow_run:
+    workflows: [ "Artifacts" ]
+    branches:
+      - master
+    types:
+      - completed
+  workflow_dispatch:
+
+jobs:
+  prepare:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    runs-on: ubuntu-20.04
+    outputs:
+      VERSION: "${{ steps.maven-project.outputs.version }}-${{ github.run_id }}-${{ github.run_number }}-${{ steps.short-sha.outputs.sha }}"
+    steps:
+      - name: Parse short sha
+        uses: benjlevesque/short-sha@v1.2
+        id: short-sha
+      - uses: actions/checkout@v2
+      - name: Get Maven project version
+        run: echo ::set-output name=version::$(mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
+        id: maven-project
+  draft-release:
+    runs-on: ubuntu-20.04
+    needs: [ prepare ]
+    env:
+      VERSION: "${{ needs.prepare.outputs.VERSION }}"
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Retrieve license obligation resources
+        run: |
+          find . -regex '.*3RD-PARTY-LICENSES' -type d -not -regex '.*target.*' | \
+          awk '{split($0,a,"3RD-PARTY-LICENSES"); system("cd " $0 " && find . -type d -maxdepth 1 -not -path . | zip -r@ \"$(cat module.name)-3rd-party-copyrights\" && cd -")}'
```

Comment:
Missing file  `module.name`.

The workflow requires a file called `module.name` located in each `3RD-PARTY-LICENSES` in order to run successfully.